pkg/operator/ceph/object/zone/zone.go

```diff
@@ -49,3 +51,27 @@ func validateZoneCR(u *cephv1.CephObjectZone) error {
 	}
 	return nil
 }
+
+// generateRandomBytes returns securely generated random bytes.
+func generateRandomBytes(length int) ([]byte, error) {
```

Comment:
Don't copy/paste code from other packages if possible, try to export the functions instead. (mgr package).